File: ios/CustomModules/CustomSwiftModule.m
//
//  UIImage+CustomSwiftModule.m
//  rn_hybrid
//
//  Created by ChunNan on 16/7/21.
//  Copyright © 2016年 Facebook. All rights reserved.
//

#import "RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(CustomSwiftModule, NSObject)

RCT_EXTERN_METHOD(call:(NSString *)name callback:(RCTResponseSenderBlock)callback)

@end

File: ios/CustomModules/CustomSwiftModule.swift
//
//  CustomSwiftModule.swift
//  rn_hybrid
//
//  Created by ChunNan on 16/7/21.
//  Copyright © 2016年 Facebook. All rights reserved.
//

import UIKit

@objc(CustomSwiftModule)
class CustomSwiftModule: NSObject {

  @objc func call(input: String, callback: RCTResponseSenderBlock) {
    print("=====>>> \(input)")
    callback(["mu ha ha ~"])
  }
  
  @objc func jumpToNative(callback: RCTResponseSenderBlock) {
    
  }
}
